x-pack/plugins/alerting/server/plugin.test.ts

```diff
@@ -74,7 +75,9 @@ describe('Alerting Plugin', () => {
           features: featuresPluginMock.createSetup(),
           unifiedSearch: autocompletePluginMock.createSetupContract(),
           // serverless setup is currently empty, and there is no mock
```

Comment:
1 / 3 - Could you remove this comment since it's obviously no longer correct :-)